pkg/manager/manager.go

```diff
@@ -113,6 +113,17 @@ type Options struct {
 	// will use for holding the leader lock.
 	LeaderElectionID string
 
+	// LeaseDuration is the duration that non-leader candidates will
+	// wait to force acquire leadership. This is measured against time of
+	// last observed ack.
+	LeaseDurationSeconds *time.Duration
```

Comment:
why are these pointers?

Reply:
Trying to keep consistent with already existing `syncPeriod`, https://github.com/kubernetes-sigs/controller-runtime/pull/412/files#diff-7d903b129bc883285ffc5068c39c00b0R102

Reply:
Please document the defaults.